common/writeThoughFile.go

```diff
@@ -25,18 +25,11 @@ import (
 	"strings"
 )
 
-func CreateParentDirectoryIfNotExist(destinationPath string) error {
+func CreateParentDirectoryIfNotExist(destinationPath string) {
 	// check if parent directory exists
 	parentDirectory := destinationPath[:strings.LastIndex(destinationPath, DeterminePathSeparator(destinationPath))]
-	_, err := os.Stat(parentDirectory)
-	// if the parent directory does not exist, create it and all its parents
-	if os.IsNotExist(err) {
-		err = os.MkdirAll(parentDirectory, os.ModePerm)
-		if err != nil {
-			return err
-		}
-	} else if err != nil {
-		return err
-	}
-	return nil
+
+	// No need to error check if the mkdir failed.
```

Comment:
@Adele My initial gut instinct is not super-comfortable with this.  In particular, if there really is an issue with making a directory, and we don't report it as such, then it becomes harder for customers to diagnose at support time. E.g. instead of getting one directory creating failure error, they get one failure per file in the log.  

Can think you think of any alternatives that don't have that draw-back?

Reply:
Could just log to the job log that the directory couldn't be created, and then log the error.